<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Qmain_actions</class>
 <widget class="QWidget" name="Qmain_actions">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>352</width>
    <height>234</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtCRMInterface</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="Qvert_layout_main">
     <item>
      <widget class="QPlainTextEdit" name="Qtext_obs">
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="Qtext_data">
       <property name="cursor" stdset="0">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="Qline">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="Qhoriz_layout_buttons" stretch="0,1,1,0">
       <item>
        <spacer name="Qhoriz_spacer_1">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="Qbutton_actions">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Actiuni</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="Qbutton_reminder">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Reminder</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="Qhoriz_spacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
